deb/index_files.go

```diff
@@ -107,6 +123,27 @@ func (file *indexFile) Finalize(signer pgp.Signer) error {
 			file.parent.renameMap[filepath.Join(file.parent.basePath, file.relativePath+file.parent.suffix+ext)] =
 				filepath.Join(file.parent.basePath, file.relativePath+ext)
 		}
+
+		if file.acquireByHash {
+			sums := file.parent.generatedFiles[file.relativePath+ext]
+
+			err = packageIndexByHash(file, ext, "SHA512", sums.SHA512)
```

Comment:
I think we can rewrite this in a loop similar line 107?

Reply:
Used map to accomplish this.